<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employeeDialog</class>
 <widget class="QDialog" name="employeeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/middle.jpg</pixmap>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>701</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1_2">
    <attribute name="title">
     <string>查询员工</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>员工号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="search_name"/>
      </item>
      <item>
       <widget class="QPushButton" name="search_submit">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>341</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_12">
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Label_67">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label_68">
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Label_69">
        <property name="text">
         <string>身份证号</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="Label_70">
        <property name="text">
         <string>家庭住址</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="Label_71">
        <property name="text">
         <string>手机号码</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="s_pwd">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="s_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="s_sex">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="s_IDnum">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="s_add">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="s_phone">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="s_admin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>是否为管理员</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage2_2">
    <attribute name="title">
     <string>添加员工</string>
    </attribute>
    <widget class="QCheckBox" name="c_admin">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>320</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>是否为管理员</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addPerson">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>391</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_11">
      <item row="1" column="0">
       <widget class="QLabel" name="Label_60">
        <property name="text">
         <string>员工号</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label_61">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="c_name"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Label_62">
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="c_sex"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="Label_63">
        <property name="text">
         <string>身份证号</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="c_IDnum"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="Label_64">
        <property name="text">
         <string>家庭住址</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="c_add"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="Label_65">
        <property name="text">
         <string>手机号码</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="c_phone"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="c_id"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="c_pwd"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>删除员工</string>
    </attribute>
    <widget class="QPushButton" name="submit_delete">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>320</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认删除</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>361</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_10">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_54">
        <property name="text">
         <string>员工号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="delete_id"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Label_55">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="request_name">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="delete_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>修改员工信息</string>
    </attribute>
    <widget class="QPushButton" name="update_clear">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>370</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submit_update">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>370</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="update_admin">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>340</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>是否为管理员</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>401</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="Label_18">
        <property name="text">
         <string>员工号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="update_id"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="update_name"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label_19">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Label_20">
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="update_sex"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="Label_21">
        <property name="text">
         <string>身份证号</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="update_IDnum"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="Label_22">
        <property name="text">
         <string>家庭住址</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="update_add"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="Label_23">
        <property name="text">
         <string>手机号码</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="update_phone"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="update_pwd"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="update_info">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>120</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>显示</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
